### PR TITLE
Added hostmetrics example to example section of README

### PR DESCRIPTION
**Pull request by asenkowski**
##### Checklist
- [x] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,76 @@ For supported exporters and their documentation see [exporters](/docs/exporters.
...
+      memory:
+      network:
+      paging:
+      processes:
+
+      # Uncomment "process" and "mute_process_name_error" to enable
+      # per process metrics for cpu, memory, and disk.
+      # This feature requires root / Administrative privileges.
+      #process:
+      #  mute_process_name_error: true
```
Comment (jsirianni) [inline README.md:109]: I think it would be okay to remove this, as its just a quick example. The full example is available in `config/`

**Review by jsirianni — COMMENTED** [review]
We have this example in `config/`. I am a little concerned that config/ could get updated and the readme could get missed.

For example, I know we will be updating all Google example configs to use `generic_node` monitored resource type.

File: README.md (hunk 1 above)
Comment (asenkowski) [reply README.md]: I agree. Removed this section and added a line at the end of the example referencing full config.

File: README.md [hunk 2]
```diff
@@ -97,6 +97,72 @@ For supported exporters and their documentation see [exporters](/docs/exporters.
 
 For supported extensions and their documentation see [extensions](/docs/extensions.md).
 
+## Example
+
+Here is an example `config.yaml` setup for hostmetrics on Google Cloud. To make sure your environment is set up with required prerequisites, see our [Google Cloud Exporter Prerequisites](/config/google_cloud_exporter/README.md) page.
```
[the remark this answers is not in the document]
Comment (cpheps) [reply README.md:102]: I think I would prefer something that says `if you're interested in sending to google cloud see our config here` with a link to the host config.

**Review by cpheps — APPROVED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (asenkowski) [reply README.md]: discussed in slack

**Review by jsirianni — APPROVED** [review]

**Pull request merged**